# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.cpp
# Commit: af89aa9ca4fb
# Full Hash: af89aa9ca4fbbaccbcd895eb33c89a8e141d7c85
# Author: stransky <stransky@redhat.com>
# Date: 2022-07-26 21:44:38
# Regressor Bug: 1777664
# File Overlap Count: 1
# Description:
#   Bug 1777664 [Linux] Destory layer manager at nsWindow::ReleaseGdkWindow() r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152694
# ==============================================================================

diff -r 0a19e7857a10 -r af89aa9ca4fb widget/gtk/nsWindow.cpp
--- a/widget/gtk/nsWindow.cpp	Tue Jul 26 14:56:49 2022 +0000
+++ b/widget/gtk/nsWindow.cpp	Tue Jul 26 14:56:49 2022 +0000
@@ -5411,9 +5411,7 @@
   WaylandStopVsync();
   DestroyChildWindows();
 
-  if (mCompositorWidgetDelegate) {
-    mCompositorWidgetDelegate->DisableRendering();
-  }
+  DestroyLayerManager();
 
   if (mGdkWindow) {
     g_object_set_data(G_OBJECT(mGdkWindow), "nsWindow", nullptr);
